CIME/SystemTests/err.py

```diff
@@ -39,9 +39,9 @@ def _case_two_custom_prerun_action(self):
         rest_root = os.path.abspath(os.path.join(dout_s_root, "rest"))
         restart_list = ls_sorted_by_fname(rest_root)
         rest_cnt = len(restart_list)
-        expect(rest_cnt >= 1, "No restart files found in {}".format(rest_root))
```

Comment:
Is this line meant to be removed? Seems like this should remain, unless it wasn't actually working anyway.